test/test_cpu/test_gguf_format.py

```diff
@@ -269,27 +269,57 @@ def test_q4_k_m(self):
         self.assertEqual(autoround.model.model.layers[10].mlp.gate_proj.bits, 8)
         self.assertEqual(autoround.layer_config['model.layers.10.mlp.gate_proj']['mostly'], "gguf:q8_0")
         shutil.rmtree("./saved", ignore_errors=True)
+
+        autoround = AutoRound(
+            model,
+            tokenizer,
+            iters=0,
+            nsamples=1,
+            seqlen=128,
+            disable_opt_rtn=False 
+        )
+        quantized_model_path = "./saved"
+        autoround.quantize_and_save(output_dir=quantized_model_path, format="gguf:q4_k_m,fake")
+        shutil.rmtree("./saved", ignore_errors=True)
     
     def test_all_format(self):
-        from auto_round.export.export_to_gguf.config import GGUF_CONFIG
-        model_name = "Qwen/Qwen2.5-7B-Instruct"
+        model_name = "Qwen/Qwen2.5-1.5B-Instruct"
         python_path = sys.executable
-        for gguf_format in GGUF_CONFIG.keys():
+        for gguf_format in ["gguf:q4_0", "gguf:q4_1", "gguf:q4_k_m", "gguf:q6_k"]:
             res = os.system(
                 f"cd ../.. && {python_path} -m auto_round --model {model_name} "
-                f" --bs 16 --iters 1 --nsamples 1 --format {gguf_format}"
+                f" --bs 16 --iters 1 --nsamples 1 --seqlen 16 --format {gguf_format}"
             )
             if res > 0 or res == -1:
                 assert False, "cmd line test fail, please have a check"
             shutil.rmtree("../../tmp_autoround", ignore_errors=True)
 
             res = os.system(
-                f"cd ../.. && {python_path} -m auto_round --model {model_name} "
-                f" --bs 16 --iters 0 --nsamples 1 --format fake,{gguf_format}"
+                f"cd ../.. && {python_path} -m auto_round --model {model_name}"
+                f" --bs 16 --iters 0 --nsamples 1 --seqlen 16 --format fake,{gguf_format}"
             )
             if res > 0 or res == -1:
                 assert False, "cmd line test fail, please have a check"
             shutil.rmtree("../../tmp_autoround", ignore_errors=True)
+    
+    def test_vlm_gguf(self):
+        model_name = "Qwen/Qwen2-VL-2B-Instruct"
```

Comment:
add more test in cuda test a ,e.g. llama, and it would be better to check the mode size